### DISCUSSION TITLE
Extent of support for Dear Imgui examples/widgets.

### DISCUSSION THREAD

**Discussion by Mo0nbase** [inkyblackness/imgui-go#164] [Q&A] [answered]

**Comment by dertseha** [comment, marked as answer]
Was the change meant to be deliberate to remove all content from the initial post?

As for the base topic: The examples project is intended to mirror the C++ implementation, yet this is a tedious task that no-one has properly taken up so far. And I'm also a bit lenient to accept pull-requests without the corresponding extension, which also is in tandem with the lack of a base infrastructure; Meaning, the examples project doesn't provide the full equivalent demo window yet for instance...
